src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -16,147 +16,13 @@
-                            }
-                        }
-                    }
-                }
-            }
-        }
-
-        private void HandleMessage(string messageTypeAsString, 
-            BlittableJsonReaderObject message, 
-            WebsocketStream webSocketStream, 
-            DocumentsOperationContext context, 
-            Guid srcDbId, 
-            IncomingDocumentReplication incomingReplication)
-        {
-            switch (messageTypeAsString)
-            {
-                case Constants.Replication.MessageTypes.GetLastEtag:
-                    using (context.OpenReadTransaction())
-                        WriteLastEtagResponse(webSocketStream, context, srcDbId);
-                    break;
-                case Constants.Replication.MessageTypes.ReplicationBatch:
-                    BlittableJsonReaderArray replicatedDocs;
-                    if (!message.TryGet(Constants.Replication.PropertyNames.ReplicationBatch, out replicatedDocs))
-                        throw new InvalidDataException(
-                            $"Expected the message to have a field with replicated document array, named {Constants.Replication.PropertyNames.ReplicationBatch}. The property wasn't found");
-                    using (context.OpenWriteTransaction())
-                    {
-                        incomingReplication.ReceiveDocuments(context, replicatedDocs);
-                        context.Transaction.Commit();
-                    }
-                    break;
-                default:
-                    throw new NotSupportedException($"Received not supported message type : {messageTypeAsString}");
-            }
-        }
-
-        //NOTE : assumes at least read transaction open in the context
-        private long GetLastReceivedEtag(Guid srcDbId, DocumentsOperationContext context)
-        {
-            var serverChangeVector = Database.DocumentsStorage.GetChangeVector(context);
-            var vectorEntry = serverChangeVector.FirstOrDefault(x => x.DbId == srcDbId);
-            return vectorEntry.Etag;
-        }
-
-        private void WriteLastEtagResponse(WebsocketStream webSocketStream, DocumentsOperationContext context, Guid srcDbId)
-        {           			
-            using (var responseWriter = new BlittableJsonTextWriter(context, webSocketStream))
-            {
-                context.Write(responseWriter, new DynamicJsonValue
-                {
-                    [Constants.Replication.PropertyNames.LastSentEtag] = GetLastReceivedEtag(srcDbId, context),
-                    [Constants.MessageType] = Constants.Replication.MessageTypes.GetLastEtag
-                });
-                responseWriter.Flush();
-            }
-        }
+    {
+		[RavenAction("/replication/topology", "GET")]
+		[RavenAction("/databases/*/replication/topology", "GET")]
+		public Task GetReplicationTopology()
```

Comment:
This caused 112 tests to fail because of duplicate route, please run the fast tests before each PR